bosh-director/lib/bosh/director/blob_util.rb

```diff
@@ -22,6 +22,15 @@ def self.copy_blob(blobstore_id)
       blobstore_id
     end
 
+    def self.replace_blob(old_blobstore_id, path)
+      blobstore.delete(old_blobstore_id) if blobstore.exists?(old_blobstore_id)
+      return create_blob path
+    end
+
+    def self.verify_blob(blobstore_id, sha1)
+      blobstore.exists?(blobstore_id) && sha1 == Digest::SHA1.hexdigest(blobstore.get(blobstore_id))
```

Comment:
here we are using the #get method of the blobstore client, and if there is no file name provided, the content will be written into a temp file and will be cleaned up by blobstore client after we get the content of the file, so here we don't manually cleanup this file